### PR TITLE
feat(Ldap Node): Add LDAP node

### PR DESCRIPTION
**Pull request by Joffcom**

### CODE REVIEW

File: packages/nodes-base/nodes/Ldap/Ldap.node.ts [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+						tlsOptions.rejectUnauthorized = credentials.allowUnauthorizedCerts === false;
+						if (credentials.caCertificate) {
+							tlsOptions.ca = [credentials.caCertificate as string];
+						}
+						if (credentials.connectionSecurity !== 'startTls') {
+							ldapOptions.tlsOptions = tlsOptions;
+						}
+					}
+
+					const client = new Client(ldapOptions);
```
Comment (maspio) [inline packages/nodes-base/nodes/Ldap/Ldap.node.ts:125]: Multiple places where we have duplicate code creating the ldap client. Would be better if we had one createClient() functon.

**Comment by madgbec** [comment]
I cannot clock any boxes

**Comment by Joffcom** [comment]
@madgbec not sure I understand, what are you trying to do?

**Comment by madgbec** [comment]
The trying to load my regpaid cards and find registeredcomponents

**Comment by Joffcom** [comment]
Are you getting any error messages  and which ldap server implementation are you using?

File: packages/nodes-base/nodes/Ldap/Ldap.node.ts (hunk 1 above)
Comment (Joffcom) [reply packages/nodes-base/nodes/Ldap/Ldap.node.ts:126]: Change is in, thanks.

**Review by maspio — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.230.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.230.1)

**Pull request merged**